tests/jax/components/executing/observing_test.py

```diff
@@ -0,0 +1,340 @@
+# python3
+# Copyright 2021 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for FeedforwardExecutorObserve class for Jax-based Mava systems"""
+
+
+from types import SimpleNamespace
+from typing import Dict, Tuple
+
+import jax.numpy as jnp
+import pytest
+from acme.types import NestedArray
```

Comment:
I think we should make an issue to remove our dependency on acme's types (I know it is not just here but in a few others places) since we have our own. Also, we should make new type definitions specific for Jax.

Reply:
I can remove it from this test since I just wanted to portray how the original function look like and change the type to Any in the functions `add` and `add_first`?

Reply:
I removed it and change the types of the parameters to Any